.idea/.gitignore

```diff
@@ -0,0 +1,7 @@
+# Default ignored files
```

Comment:
let's remove all the `.idea` files and add the `.idea` directory to `.gitignore`